### PR TITLE
Changed DashboardTodo code to closer match JournalTodo

### PR DESCRIPTION
**Pull request by AngelaLH**
Progress on #118 

## Proposed Changes

Features updated to closer match JournalTodo:
- Variables/functions with identical functionality changed to match
- "Migrate/Cancel/Delete" Menu 
- "Migrate" dialog
- "New to do" dialog

Next Steps:
- Refactor identical code out of Journaltodo and Dashboardtodo into a new file
- Update the style of dashboard todo to match journal todo

### CODE REVIEW

**Review by a-158w — APPROVED** [review]
This looks really good and it will be nice to have a more consistent code design for both the todo views! The changes all look good to me!

**Review by yzho976 — COMMENTED** [review]
Very nice formatting and refactoring, but just I thing I want to clarify;

File: telos-frontend/src/components/todo/DashboardTodo.jsx [hunk 1]
```diff
@@ -53,40 +53,40 @@ const outdated = {
 const DashboardTodo = () => {
   const listitems = [
     {
-      name: 'OnGoing',
+      name: 'isOverdue',
       due: '2021-04-30',
-      onGoing: true,
+      isOverdue: true,
       completed: false,
```
Comment (yzho976) [inline telos-frontend/src/components/todo/DashboardTodo.jsx:59]: I think onGoing and isOverdue have opposite meanings, is it okay to change the name to a contracting word without changing the state?

Comment (ambiyu) [reply telos-frontend/src/components/todo/DashboardTodo.jsx:59]: The backend uses isOverdue, so I think we should stick with it to be consistent. I think isOverdue makes more sense than onGoing either way.
But yes, if you change from onGoing to isOverdue, you must switch the conditions around to keep everything the same. Could you please make that change?

File: telos-frontend/src/components/todo/DashboardTodo.jsx [hunk 2]
```diff
@@ -108,7 +108,12 @@ const DashboardTodo = () => {
   // Change date for reschedule
   const reDateChange = (event) => {
     setReDate(event.target.value);
-    setReItem({ name: reItem.name, due: event.target.value, onGoing: true, completed: false });
+    setselectedTodo({
+      name: selectedTodo.name,
+      due: event.target.value,
+      isOverdue: true,
```
Comment (ambiyu) [inline telos-frontend/src/components/todo/DashboardTodo.jsx]: isOverdue should be false here after the name change

File: telos-frontend/src/components/todo/DashboardTodo.jsx [hunk 3]
```diff
@@ -123,22 +128,27 @@ const DashboardTodo = () => {
...
 
-  const closeAdd = () => {
-    setAdd(false);
-    sortEvent();
+  const dateChange = (event) => {
+    setTodoDueDate(event.target.value);
+    setTodoName({
+      name: todoName.name,
+      due: event.target.value,
+      isOverdue: true,
```
Comment (ambiyu) [inline telos-frontend/src/components/todo/DashboardTodo.jsx]: isOverdue: false

File: telos-frontend/src/components/todo/DashboardTodo.jsx [hunk 4]
```diff
@@ -198,43 +200,43 @@ const DashboardTodo = () => {
...
+              className={styles.tasks}
+              key={todo.name}
               role={undefined}
               dense
               button
-              onClick={handleToggle(value.name)}
+              onClick={handleToggle(todo.name)}
             >
-              {value.onGoing ? (
+              {todo.isOverdue ? (
```
Comment (ambiyu) [inline telos-frontend/src/components/todo/DashboardTodo.jsx]: Should be change to !todo.isOverdue

File: telos-frontend/src/components/todo/DashboardTodo.jsx [hunk 5]
```diff
@@ -243,45 +245,47 @@ const DashboardTodo = () => {
...
+                    color="primary"
+                    checked={todo.completed}
                     tabIndex={-1}
                     disableRipple
                     inputProps={{ 'aria-labelledby': labelId }}
                   />
                 </ListItemIcon>
               )}
-              {value.onGoing ? (
+              {todo.isOverdue ? (
```
Comment (ambiyu) [inline telos-frontend/src/components/todo/DashboardTodo.jsx]: Should be change to !todo.isOverdue

File: telos-frontend/src/components/todo/DashboardTodo.jsx (hunk 1 above)
Comment (AngelaLH) [reply telos-frontend/src/components/todo/DashboardTodo.jsx:59]: I see what you mean, I changed it to match JournalTodo, but I am not sure what the existing state names are now
 @ambiyu thoughts?

**Review by ambiyu — CHANGES_REQUESTED** [review]
Looks good. Please the following changes to the conditions with isOverdue.

**Review by ambiyu — APPROVED** [review]

**Pull request merged**